VERSION_OMICRON

```diff
@@ -1 +1 @@
-rel/v12/rc1
\ No newline at end of file
+e036c80
```

Comment:
Dumb question, why bump to this version now instead waiting for R13 RC?

Reply:
Not dumb at all! Since we release the terraform provider immediately after this one, I like to have it updated and stable to be ready to go when the tag is made.